docs/modules/trino/pages/usage-guide/operations/pod-disruptions.adoc

```diff
@@ -0,0 +1,58 @@
+= Allowed Pod disruptions
+
+You can configure the allowed Pod disruptions for Trino nodes as described in xref:concepts:operations/pod_disruptions.adoc[].
+
+Unless you configure something else or disable our PodDisruptionBudgets (PDBs), we write the following PDBs:
+
+== Coordinators
+We only allow a single coordinator to be offline at any given time, regardless of the number of replicas or `roleGroups`.
+
+== Workers
+Normally users deploy multiple workers to speed up queries, handle multiple queries in parallel or to just have enough memory available in the Cluster to execute a big query.
+
+Taking this into consideration, our operator uses the following algorithm to determine the maximum number of workers allowed to be unavailable at the same time:
+
+`num_workers` is the number of workers in the Trino cluster, summed over all `roleGroups`.
+
+[source,rust]
+----
+// As users normally scale Trino workers to achieve more performance, we can safely take out 10% of the workers.
+let max_unavailable = num_workers / 10;
+
+// Clamp to at least a single node allowed to be offline, so we don't block Kubernetes nodes from draining.
+let max_unavailable = max(max_unavailable, 1)
+----
```

Comment:
Not sure we want rust code in the docs? This could be said with one sentence but im fine with both. @fhennig any opinions?

Reply:
My tough process was to describe the algorithm, so users can calculate maxUnavailable themselves, e.g. to decided whether to scale up or not. The reasoning how we come to the number should be documented in the code as well as in the docs, so why not copy/past them to avoid things getting out of sync :)

But no strong opinion, we could also link to the file I guess